src/bubble_chart/bubble_chart_container.js

```diff
@@ -120,13 +131,6 @@ looker.plugins.visualizations.add({
       });
       return;
     }
-    if (data.length === 0) {
-      this.addError({
-        title: "No Results",
-      });
-      done();
```

Comment:
Calling done() after adding the error is what caused the bug.  Visualization would return but it would never add the error message it wouldn't update or show.